x-pack/plugins/cases/public/components/user_actions/comment/registered_attachments.tsx

```diff
@@ -122,8 +129,15 @@ export const createRegisteredAttachmentUserActionBuilder = <
         timelineAvatar: attachmentViewObject.timelineAvatar,
         actions: (
           <>
-            <UserActionCopyLink id={comment.id} />
-            {attachmentViewObject.actions}
+            <UserActionContentToolbar
+              actions={['delete']}
+              id={comment.id}
+              deleteLabel={i18n.DELETE_COMMENT}
+              deleteConfirmTitle={i18n.DELETE_COMMENT_TITLE}
+              isLoading={isLoading}
+              onDelete={handleDeleteComment.bind(null, comment.id)}
```

Comment:
Just curious, when do we need to bind a function? Is this needed since we're passing `null` for the `this` object as the first parameter?

Reply:
Nope 🙂 . This is a pattern I copied from another file. Fixed.